### ISSUE TITLE
Help format issue w/ propsLong

### ISSUE THREAD

**Issue by CSAllisonPrecisely** [scallop/scallop#172]
This configuration:

```
class PropsLongBug(arguments: Seq[String]) extends ScallopConf(arguments) {
	val props = propsLong[String]()
	verify()
}
```

Yields help output of:

```
      --Propskey=value [key=value]...
  -h, --help                            Show help message
```

However, it should be (note the space after --Props):

```
      --Props key=value [key=value]...
  -h, --help                            Show help message

```

Execution works as expected, i.e. this input parses:

`new PropsLongBug(Seq("--Props", "key1=value1", "key2=value2"))`

But this fails:

`new LongPropsBug(Seq("--Propskey1=value1", "key2=value2"))`

Tested with 3.1.2

**Comment by Rogach** [comment]
Sorry for the late reply. Yes, that's a bug, I'll fix it now.

**Comment by Rogach** [comment]
Fixed, will release `3.1.3` in an hour.
